src/huggingface_hub/repository.py

```diff
@@ -546,6 +644,8 @@ def auto_track_large_files(self, pattern="."):
         # Cleanup the .gitattributes if files were deleted
         self.lfs_untrack(deleted_files)
 
+        return files_to_be_staged
```

Comment:
Returning the files to be staged to the user so as to warn them that the `add` operation might take a bit of time.